Controllers/UserController.cs

```diff
@@ -97,6 +126,73 @@ public async Task<IActionResult> Signup(RegisterDTO model)
             return Created("", new { token });
         }
 
+        [HttpPost("userstatus")]
+        public async Task<IActionResult> UserStatus(StatusDTO model) {
+            if (!IsValidApiRequest())
+            {
+                return ApiBadRequest("Invalid Headers!");
+            }
+            Employee user = Context.Employees.Where(z => z.Id == model.Id).FirstOrDefault();
```

Comment:
We should use `var` keyword for method variables. Also this LINQ could be simplified as `Context.Employees.FirstOrDefault(z => z.Id == model.Id);`